db/migrate/20150910010112_create_team_memberships.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTeamMemberships < ActiveRecord::Migration
+  def change
+    create_table :team_memberships do |t|
+      t.belongs_to :team
+      t.belongs_to :user
+      t.integer :role, default: 0
```

Comment:
I was just transferring what was there. I think the nitty-gritty details are hard to decide on before implementing some use cases.